### PR TITLE
Make the code compatible with Python 3

### PR DESCRIPTION
**Pull request by oneeyeman1**
The code is currently run under python 3 as well as it should continue working under python 2.

Please review and apply.

### CODE REVIEW

**Comment by vadz** [comment]
I think this should be fine, AFAIK only `print(foo, bar)` works differently from `print foo, bar` in Python 2, but in this script print is only used with a single argument, so putting parentheses around it should be harmless for Python 2. Please correct me if I'm wrong.

File: misc/scripts/png2c.py [hunk 1]
```diff
@@ -20,7 +20,7 @@
   -s    embed the image size in the image names in generated code."""
 
 if len(sys.argv) < 2:
-        print USAGE
+        print( USAGE )
```
Comment (swt2c) [inline misc/scripts/png2c.py]: Stylistic comment - I would get rid of the extra spaces you have around the arguments in all of these calls.  It's not typically how Python is stylized and not consistent with the rest of the file.

Comment (oneeyeman1) [reply misc/scripts/png2c.py]: Fixed with the latest commit.

Comment (swt2c) [reply misc/scripts/png2c.py]: LGTM.

Comment (oneeyeman1) [reply misc/scripts/png2c.py]: @swt2c, thx.

**Comment by oneeyeman1** [comment]
@vadz , The porting to Python 3 is complete.
Please review and apply.

**Comment by oneeyeman1** [comment]
@swt2c ,
Please review latest commit.

Thx.

**Comment by swt2c** [comment]
Looks OK to me.  I wasn't familiar with the `array` module, but just looked at it.  What's somewhat confusing is that `array.tostring()` actually returns bytes.  They've renamed `tostring()` to `tobytes()` in Python 3.2 I think but they kept the old name for compatibility.

**Comment by oneeyeman1** [comment]
Hi,
> 
> 
> Looks OK to me. I wasn't familiar with the `array` module, but just looked at it. What's somewhat confusing is that `array.tostring()` actually returns bytes. They've renamed `tostring()` to `tobytes()` in Python 3.2 I think but they kept the old name for compatibility.

Thank you for the review.
Let's wait for Vadim (or Robin) to check and apply.

**Comment by vadz** [comment]
The use of `b"..."` makes the code incompatible with Python version before 2.6. I don't necessarily think we should care about such ancient versions, but I wanted to add something like
```python
if sys.hexversion < 0x02060000:
        print("Sorry, this script requires Python 2.6 or later (including Python 3).")
        sys.exit(2)
```
to at least avoid nonsensical errors, but this doesn't work, as Python 2.5 refuses to even compile the script using `b"..."` notation. I don't know if there is any simple way to do something like I wanted, but for now I'll just merge it as is.

**Pull request closed without merging**